docs/source/reference/leaderboard_faq.rst

```diff
@@ -0,0 +1,132 @@
+
+The tests are run from different regions in order to assess the transcoding performance a user in that region can expect from orchestrators.
+
+The regions that transcoding tests are run from are subject to change in order to asess orchestrator performance from a greater number of regions in the world.
+
+What metrics are displayed in the leaderboard?
+************************************************
+
+The leaderboard currently displays the following metrics:
+
+- **Total score**: The average utility of using an orchestrator based on past transcoding tests.
+- **Success rate**: The average percentage of source video segments that were successfully transcoded in past transcoding tests.
+- **Latency score**: The average utility of an orchestrator's overall transcoding latency in past transcoding tests.
+
+Additional metrics for other aspects of an orchestrator's performance may be added in the future.
+
+How is the success rate in the leaderboard calculated?
+********************************************************
+
+The success rate is calculated by counting the number of transcoded results (could contain one or many output renditions) received from an orchestrator and dividing it by the number of source video segments in a stream.
+For example, given 100 source video segments in a stream and 60 transcoded results received, the success rate would be 60%.
+
+The number of transcoded results received in a transcoding test is impacted by whether the orchestrator is online during the test and whether the orchestrator can 
+keep up with the stream. The latter is also influenced by the broadcaster's implementation. The current broadcaster implementation results in dropped segments (missing transcoded results)
+if a) the orchestrator is still transcoding the previous segment when a new segment is ready or b) the orchestrator takes longer than 8 seconds to return the transcoded results
+for a segment.
+
+How is the latency score in the leaderboard calculated?
+*********************************************************
+
+The current formula for the latency score is:
+
+::
+
+    latency_score = 1 - e^{-base_latency_score}
+    base_latency_score = avg_segment_duration / avg_round_trip_time
+
+:code:`base_latency_score` measures the average round time for a broadcaster to receive the transcoded results for a segment relative to the average segment duration in a stream. When this value is less than
+1.0, the orchestrator is returning transcoded results slower than real-time. When the value is greater than or equal to 1.0, the orchestrator is returning transcoded results
+in real-time or faster. 
+
+Since live streaming applications require fast transcoding, the latency score increases as the base latency score increases. However, the rate of increase for the latency score 
+also decreases as the base latency score increases which reflects diminishing utility from increases in transcoding speed.
+
+How is the total score in the leaderboard calculated?
```

Comment:
WDYT about adding a note at the bottom of this section that the leaderboard displays this on a scale of 0-10 ?

Reply:
Fixed in c2055f4